app/src/main/java/com/sun/wsm_android_v2/data/model/User.kt

```diff
@@ -0,0 +1,46 @@
+package com.sun.wsm_android_v2.data.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+class User {
+    @SerializedName("name")
+    @Expose
+    var name: String? = null
+    @SerializedName("email")
+    @Expose
+    var email: String? = null
+    @SerializedName("employee_code")
+    @Expose
+    var employeeCode: String? = null
+    @SerializedName("birthday")
+    @Expose
+    var birthday: String? = null
+    @SerializedName("start_probation_date")
+    @Expose
+    var startProbationDate: String? = null
+    @SerializedName("end_probation_date")
+    @Expose
+    var endProbationDate: String? = null
+    @SerializedName("contract_date")
+    @Expose
+    var contractDate: String? = null
+    @SerializedName("gender")
+    @Expose
+    var gender: String? = null
+    @SerializedName("avatar_url")
+    @Expose
+    var avatarUrl: Any? = null
```

Comment:
url avatar sẽ là String đó e
```suggestion
    var avatarUrl: String? = null
```